### PR TITLE
Fix release instructions in contributing docs

### PR DESCRIPTION
**Pull request by laurieontech**
Updating release instructions per conversation with @johno

### CODE REVIEW

**Comment by johno** [comment]
Thanks!

File: docs/contributing.mdx [hunk 1]
```diff
@@ -126,11 +126,7 @@ MDX is a monorepo that uses [lerna][].
 In order to release a new version you can follow these steps:
 
 - Draft a release for the next version (vX.X.X)
-- Release a prerelease
-  - `yarn lerna publish`
-  - Select prepatch/preminor/premajor
-  - Sanity check in a project or two with the prerelease
-- `yarn lerna publish`
+- `yarn && yarn test && yarn yarn run publish`
```
Comment (wooorm) [inline docs/contributing.mdx:129]: @laurieontech `yarn yarn (run publish)`? Is that a thing 😅

Comment (laurieontech) [reply docs/contributing.mdx:129]: Ha! We both missed it. Will fix again.

**Comment by silvenon** [comment]
Thanks for this! I tried to publish first with `yarn publish` and was confused for a while 😅

**Pull request merged**